packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -236,12 +256,17 @@ export const ChannelView: React.FC = () => {
     <ViewWrapper>
       <ChannelCover assetUrl={coverPhotoUrl} />
       <LimitedWidthContainer>
+        {smMatch ? (
+          <CollectorsBoxContainer>
+            {collectors.length > 0 && <CollectorsBox collectors={collectors} />}
```

Comment:
I still don't like the fact we have this empty div that just taking the space 😅 
![image](https://user-images.githubusercontent.com/51168865/151518285-4762b839-459f-4be0-9e98-5c3dd7d7c7cc.png)
I mean, I don't have a better solution for this, so if you don't come up with an idea of how to remove this guy, just leave it as it is.

Reply:
I don't have an issue with it it just keeps the space for the collector box, a solution for it would be more complicated than just not doing nothing